app/models/portfolio_item.rb

```diff
@@ -18,4 +18,16 @@ class PortfolioItem < ApplicationRecord
 
   belongs_to :portfolio
   validates :service_offering_ref, :presence => true
+
+  def resolved_workflow_refs
+    # TODO: Add lookup to platform workflow ref
+    # https://github.com/ManageIQ/catalog-api/issues/164
+    [item_workflow_ref]
```

Comment:
Since `item_workflow_ref` potentially returns `nil` you will want to `compact!` this array.